requirements.txt

```diff
@@ -21,3 +21,4 @@ PyYAML==3.11
 pint>=0.7
 numpy
 functools32
+elasticsearch>=2.0.0,<3.0.0
```

Comment:
@tedhtchang I don't think it's worth adding the file for a single module. If elasticsearch eventually requires several new modules to be installed, then we can add a requirements file.